### PR TITLE
[zh-cn]: update translation of HTTP `Access-Control-Allow-Methods` header

### PR DESCRIPTION
**Pull request by T34-active**

### CODE REVIEW

File: files/zh-cn/web/http/headers/access-control-allow-methods/index.md [hunk 1]
```diff
@@ -1,31 +1,46 @@
...
+
+<table class="properties">
+  <tbody>
+    <tr>
+      <th scope="row">标头类型</th>
+      <td>{{Glossary("Response header", "响应标头")}}</td>
+    </tr>
+    <tr>
+      <th scope="row">{{Glossary("Forbidden header name", "禁止修改的标头")}}</th>
+      <td>无</td>
```
Comment (T34-active) [inline files/zh-cn/web/http/headers/access-control-allow-methods/index.md:20]:
```suggestion
      <td>否</td>
```

File: files/zh-cn/web/http/headers/access-control-allow-methods/index.md [hunk 2]
```diff
@@ -1,31 +1,46 @@
...
+```http
+Access-Control-Allow-Methods: <method>, <method>, …
+Access-Control-Allow-Methods: *
 ```
 
 ## 指令
 
 - \<method>
-  - : 用逗号隔开的允许使用的 [HTTP request methods](/zh-CN/docs/Web/HTTP/Methods) 列表。
+  - : 一个以逗号分隔的允许 [HTTP 请求方法](/zh-CN/docs/Web/HTTP/Methods)的列表。
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/access-control-allow-methods/index.md:35]: 改进翻译：

```suggestion
  - : 一个以逗号分隔，表示允许使用的 [HTTP 请求方法](/zh-CN/docs/Web/HTTP/Methods)的列表。
```

File: files/zh-cn/web/http/headers/access-control-allow-methods/index.md [hunk 3]
```diff
@@ -1,31 +1,46 @@
...
+- `*` （通配符）
+  - : “`*`”值仅在无凭证的请求（即不包含 [HTTP cookie](/zh-CN/docs/Web/HTTP/Cookies) 或 HTTP 认证信息的请求）中视为特殊的通配符值。在带有凭证的请求中，它被当作字面意义的标头名称“`*`”处理，不具有特殊语义。
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/access-control-allow-methods/index.md:37]:
```suggestion
  - : “`*`”值仅在无凭据的请求（即不包含 [HTTP cookie](/zh-CN/docs/Web/HTTP/Cookies) 或 HTTP 认证信息的请求）中视为特殊的通配符值。在带有凭据的请求中，它被当作字面意义的标头名称“`*`”处理，不具有特殊语义。
```

**Review by yin1999 — APPROVED** [review]
:+1:

File: files/zh-cn/web/http/headers/access-control-allow-methods/index.md [hunk 4]
```diff
@@ -1,31 +1,46 @@
...
+Access-Control-Allow-Methods: <method>, <method>, …
+Access-Control-Allow-Methods: *
 ```
 
 ## 指令
 
 - \<method>
-  - : 用逗号隔开的允许使用的 [HTTP request methods](/zh-CN/docs/Web/HTTP/Methods) 列表。
+  - : 一个以逗号分隔，表示允许使用的 [HTTP 请求方法](/zh-CN/docs/Web/HTTP/Methods)的列表。
+- `*` （通配符）
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/access-control-allow-methods/index.md:36]:
```suggestion
- `*`（通配符）
```

**Pull request merged**